### PR TITLE
update doc about image

### LINKED ISSUE #35: update document about docker image
Now the docker image is not found at cargo.caicloud.io, we should fix it immediately.

Due: 11.9

### PR DESCRIPTION
**Pull request by gaocegege**
fixes #35 

Signed-off-by: Ce Gao <ce.gao@outlook.com>

### CODE REVIEW

File: docker-compose-dev.yml [hunk 1]
```diff
@@ -1,47 +1,45 @@
...
-    container_name: clair_postgres
-    image: index.caicloud.io/caicloud/postgres:9
+    image: postgres:9
     environment:
       POSTGRES_PASSWORD: password
 
 clair:
-    container_name: clair_clair
-    image: index.caicloud.io/caicloud/clair:v1.2.3
+    image: quay.io/coreos/clair:v1.2.3
```
Comment (superxi911) [inline docker-compose-dev.yml]: change to cargo.caicloud.io/caicloud/clair:v1.2.3

Comment (gaocegege) [inline docker-compose-dev.yml]: OK

**Comment by gaocegege** [comment]
test tasks:
- [ ] docker-compose.yaml
- [ ] local-up
- [ ] e2e-test
- [ ] run cyclone on k8s

**Comment by gaocegege** [comment]
Could @superxi911 or @xiaomuqiao or @zoumo run the other two tests? Because I have no k8s cluster and the linux machine = =

**Comment by superxi911** [comment]
I'll deal with these.

**Comment by xiaomuqiao** [comment]
我来看 run cyclone on k8s

File: docker-compose-dev.yml [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
-
-etcd-server:
-    image: index.caicloud.io/caicloud/circle-etcd:v2.2.1
+    expose:
+        - "9092"
+etcd:
+    image: cargo.caicloud.io/caicloud/etcd:v2.2.1
     volumes:
-        - /data/etcd:/data
+        - /data:/data
```
Comment (superxi911) [inline docker-compose-dev.yml]: needn't volume

File: docker-compose-dev.yml [hunk 3]
```diff
@@ -51,3 +51,11 @@ etcd-server:
         - name=etcd-server
         - advertise-client-urls=http://0.0.0.0:4001
         - initial-advertise-peer-urls=http://0.0.0.0:7001
+
+mongo:
+    image: mongo:3.0.5
+    volumes:
+       - /data/db:/data/db
```
Comment (superxi911) [inline docker-compose-dev.yml]: needn't volume

**Comment by superxi911** [comment]
Need to delete this not used var
https://github.com/gaocegege/cyclone/blob/update-doc/tests/version/version_test.go#L49

**Comment by superxi911** [comment]
LGTM

**Comment by xiaomuqiao** [comment]
I make some changes to support setup cyclone into kubernetes, PTAL

File: scripts/k8s/postgres.yaml [hunk 4]
```diff
@@ -24,7 +24,7 @@ spec:
       - env:
         - name: POSTGRES_PASSWORD
           value: password
-        image: postgres:9
+        image: cargo.caicloud.io/caicloud/postgres:9
```
Comment (gaocegege) [inline scripts/k8s/postgres.yaml]: By the way, could you update the docker-compose.yml and docker-compose-dev.yml?

:)

File: docs/setup_zh-CN.md [hunk 5]
```diff
@@ -33,21 +33,22 @@
...
-	kubectl create -f etcd.yaml
-	kubectl create -f etcd-svc.yaml
-	kubectl create secret generic clairsecret --from-file=config.yaml
-	kubectl create -f postgres.yaml
-	kubectl create -f postgres-svc.yaml
-	kubectl create -f clair.yaml
-	kubectl create -f clair-svc.yaml
-	kubectl create -f cyclone.yaml
-	kubectl create -f cyclone-svc.yaml
+    kubectl create namespace cyclone
```
Comment (gaocegege) [inline docs/setup_zh-CN.md]: \t and four spaces are mixed

:)

**Comment by gaocegege** [comment]
OK I would fix the problem what I reviewed.

**Pull request merged**